src/ol_infrastructure/applications/learn_ai/Pulumi.applications.learn_ai.CI.yaml

```diff
@@ -29,7 +29,7 @@ config:
   learn_ai:frontend_vars:
     MIT_LEARN_APP_BASE_URL: https://rc.learn.mit.edu/
     OPENEDX_API_BASE_URL: https://courses-qa.mitxonline.mit.edu/
-    OPENEDX_LOGIN_URL: https://rc.mitxonline.mit.edu/signin/
+    OPENEDX_LOGIN_URL: https://courses-qa.mitxonline.mit.edu/auth/login/ol-oauth2/?auth_entry=login
```

Comment:
the original login url used here doesn't establish a session in openedx (unless passed a relevant `?next` parameter, which we don't have on the sandbox).